code_to_gast/routers/js_router.py

```diff
@@ -41,6 +41,8 @@ def node_to_gast(node):
     #statements
     elif node.type == "VariableDeclaration":
         return js_assign.assign_to_gast(node)
+    elif node.type == "AssignmentExpression":
+        return js_assign.aug_assign_to_gast(node)
```

Comment:
It looks like ["AssignmentExpressions"](https://esprima.readthedocs.io/en/4.0/syntax-tree-format.html?highlight=AssignmentExpression#assignment-expression) include regular `=` signs is that going to interfere with statements like `const x = 5`?

And if not... maybe `aug_assign_to_gast` should be renamed to `assignment_expression_to_gast`

Reply:
Statements like the one you mentioned fall under variable declaration in JS. A reassignment like simply x = 5 would be a AssignmentExpression and that would be translated into Python correctly because python treats all variable assignments the same.